### DISCUSSION TITLE
Grid gap feature

### DISCUSSION THREAD

**Discussion by ghost** [katoid/angular-grid-layout#43] [Q&A] [answered]
Great work with this library!

 I'm looking forward to the grid gap feature. Do you have an expected implementation date and a possible (css) work around in the mean time? I would like to have a 5px gap between the grid items.
 
 Thanks!

**Comment by llorenspujol** [comment]
Thanks! 😄 

It is on the roadmap and it is not a difficult issue to pull of, though I don't know exactly when it would be implemented, since there haven't been much demand since now:).

In the meantime, you can set a padding of 2.5px on each grid item, with that, you will gain a gap of 5px. The counter part is that those 2.5 pixels would be added also on the corners of the grid, but is not big issue.
Try this code:
```css
.ktd-grid-item {
    padding: 2.5px;
    box-sizing: border-box;
}

```

**Comment by arnoudb** [comment]
Hi, i saw this great  library and it has my preference above gridster. Amazing work!

So we would like to use it in our project but have 2 issues. One is being solved at the moment (support of gap). The other is that our project is angular v13. So the question would be: Will the next release that supports gaps also run on angular v13? Forks are due to the company policies not an option.

**Reply by llorenspujol** [reply]
I am not 100% sure, but I think that an angular v14 build is not compatible with Angular v13, so you would need to upgrade your project to Angular v14 to use this feature.
Alternatively, you can use the `padding` on `ktd-grid-item` solution, that is not that bad after all.
Forks are not usually a good option... the best option is to upgrade your project to Angular v14. If not, you can copy paste all the code, an make a ktd-grid in your project and built it with angular v13. When you upgrade to Angular v14, you could  remove the code in your project and use the library.

**Reply by arnoudb** [reply]
Thank you for the fast reply! We can go with the the padding solution indeed until we upgrade to angular 14. Done some tests and that works fine. Temporary copying the code inside the project is also a possibility indeed as I didn't see any angular 14 specific things. Nice that you confirm that. Thnx again for your work on this great project!

**Comment by llorenspujol** [comment, marked as answer]
Grid gap feature is solved and available since the version 2.0.0 of angular-grid-layout (with Angular v14): https://github.com/katoid/angular-grid-layout/pull/60
